### ISSUE TITLE
Exclude origin from header

### ISSUE THREAD

**Issue by juppjupp** [kerryjiang/WebSocket4Net#132]
Hi!
Is there anyway to exclude origin from header? The LG tv websocket Im trying to connect to does not support origin in header. Will just get error "invalid origin".

**Comment by kerryjiang** [comment]
Probably, they just need a specific origin value? You can pass it from the constructor.

**Comment by juppjupp** [comment]
Yes i know that i can pass origin from constructor. But the problem is that they dont want origin as header at all :) Therefore i want to exclude it.
